gw_eccentricity/load_data.py

```diff
@@ -161,7 +165,10 @@ def load_waveform(origin="LAL", **kwargs):
         - "LAL": Compute waveform by a call to the LAL-library.
             (see https://lscsoft.docs.ligo.org/lalsuite/lalsimulation/)
         - "SXSCatalog": Import waveform by reading a file in the SXS catalog
-            format.
+            in the new format (from 2023 onward).
```

Comment:
Be more helpful, say what new and old means in terms of the h5 filenames.

Reply:
Done